### PR TITLE
[DEV-3389] Self serve domains visual-dev updates

### PR DESCRIPTION
**Pull request by noahwc**
## Description of the change

> Description here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: [DEV-3389](https://saasquatch.atlassian.net/browse/DEV-3389)
- Process.st launch checklist: https://app.process.st/runs/Visual-Dev-150-t5e5pF0JZXKf8MeV0YVNpQ/tasks/ipJPSwLUytFe3xMt6P5PYw

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has a Jira number
- [x] This pull request has a Process.st launch checklist

### Code review

- [] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

[DEV-3389]: https://saasquatch.atlassian.net/browse/DEV-3389?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/visual-dev/CHANGELOG.md [hunk 1]
```diff
@@ -25,14 +25,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Components changed
   - \<RadioCard>
-    - Refactored `RadioCardGroupView` to now use dot notation `RadioCardView.GroupView`
-    - Updated title & description props to now allow for `string | React.ReactNode`
-    - Removed unnecessary ternary conditionals
-    - Changed `min-height` from `138px` to `95px`
+    <<<<<<< HEAD - Added more flexibility to allow custom CSS and better design for cards without icons - Added a disabled state
```
Comment (noahwc) [inline packages/visual-dev/CHANGELOG.md:28]: Fix git conflict in changelog

**Comment by noahwc** [comment]
Needs a changelog, otherwise, I'm happy

File: packages/visual-dev/CHANGELOG.md (hunk 1 above)
Comment (zachharrison) [inline packages/visual-dev/CHANGELOG.md:28]: We also need to make sure that we remember to add git tags at the bottom of the CHANGELOG

File: packages/visual-dev/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/visual-dev",
-  "version": "1.4.0",
+  "version": "1.5.0-3",
```
Comment (zachharrison) [inline packages/visual-dev/package.json:3]: Need to remember to add git tags for this new version at the bottom of the CHANGELOG

**Review by zachharrison — APPROVED** [review]
Code looks good to me.

**Review by zachharrison — APPROVED** [review]

**Pull request merged**